ot-node.js

```diff
@@ -201,11 +201,12 @@ class OTNode {
         this.logger.info('Event emitter initialized');
     }
 
-    async initializeBlockchainEventListenerService() {
+    async initializeBlockchainEventListenerModule() {
         try {
-            const eventListenerService = this.container.resolve('blockchainEventListenerService');
-            await eventListenerService.initialize();
-            eventListenerService.startListeningOnEvents();
+            const eventListenerModule = this.container.resolve('eventListenerModuleManager');
+            const implementation = await eventListenerModule.getImplementation();
```

Comment:
Can all of initialization be moved to modulManager maybe?

Reply:
Yes, it's a good idea

Reply:
Modules should be independent of each other. There is a reason why this was made into a service and not a module in the first place, as the blockchain event listener depends on both blockchain and repository modules.
All modules should be initialized in the "initializeModules" function, and module managers should not contain any logic, they are just interfaces. The idea was that modules could one day be separate npm modules.
I would strongly reconsider these changes.